src/mcts/params.cc

```diff
@@ -53,6 +53,9 @@ const OptionId SearchParams::kLogitQId{
     "logit-q", "LogitQ",
     "Apply logit to Q when determining Q+U best child. This makes the U term "
     "less dominant when Q is near -1 or +1."};
+const OptionId SearchParams::kOneMinusEpsId{
+    "one-minus-eps", "OneMinusEps",
+    "Scaling factor to apply before computing logit."};
```

Comment:
OneMinusEps makes sense as a constant but not as an option. Maybe rename it to something like LogitScale?